docker-compose/camunda-8.6/README.md

```diff
@@ -152,31 +156,27 @@ If you enabled authentication for GRPC requests on Zeebe you need to provide cli
 
 ## Web Modeler Self-Managed
 
-> :information_source: Web Modeler Self-Managed is available to Camunda enterprise customers only.
+> :information_source: Web Modeler Self-Managed images with version <8.6.0-alpha2 are available to Camunda enterprise customers only. Follow the instructions in the root folder [README](../../README.md) to set up the environment for these versions.
 
-The Docker images for Web Modeler are available in a private registry. Enterprise customers either already have credentials to this registry, or they can request access to this registry through their CSM contact at Camunda.
+Web Modeler can be run standalone with only Identity and Keycloak as dependencies.
 
-To run Camunda Platform with Web Modeler Self-Managed clone this repo and issue the following commands:
+Issue the following commands to only start Web Modeler and its dependencies:
 
 ```
-$ docker login registry.camunda.cloud
-Username: your_username
-Password: ******
-Login Succeeded
-$ docker compose --profile web-modeler up -d
+docker compose --profile web-modeler-only up -d
```

Comment:
❓ Would it make sense to name the profiles after their use cases? Eg. instead of `web-modeler-only` we could use `modeling`

Reply:
I think that is better! Is `full` for the full experience still a good name then?

Reply:
> Is `full` for the full experience still a good name then?

I think `full` is fine for now 👍